tests/test_pythonpackage_basic.py

```diff
@@ -340,6 +302,10 @@ def test_venv(self):
                 os.path.join(test_dir, "venv", "bin", "pip"),
                 "install", "-U", "pep517<0.7.0"
             ])
+            subprocess.check_output([
+                os.path.join(test_dir, "venv", "bin", "pip"),
+                "install", "-U", "toml"
+            ])
```

Comment:
This is a bit out of scope of this PR, but I'm thinking why don't we `pip install -e` then?

Reply:
Well I suppose that you are speaking of `pip install -e .` right?

In this case...why should we? we don't need all p4a dependencies for this test, and the venv will be destroyed after is passed. Also, considering that we are speaking of one of the slowest tests of p4a, IMHO, better to try to save some time :wink:

Reply:
makes sense :+1: